.github/workflows/package-helm.yaml

```diff
@@ -30,10 +35,79 @@ jobs:
             mv $SRC_FILE $DEST_FILE
           done
 
-      - name: Upload Helm packages to GitHub releases
+      - name: Upload Release Helm Chart Packages to GitHub Release
         uses: softprops/action-gh-release@v1
         with:
           name: ${{ github.ref_name }}
           draft: true
           append_body: true
           files: nri-*helm-chart*.tgz
+
+  unstable:
+    concurrency:
+      group: gh-pages
+      cancel-in-progress: false
+    if: ${{ github.env.IS_RELEASE != 'true' }}
+    permissions:
+      contents: write
```

Comment:
to avoid duplication, I would set it at the root level
```

name: Package Helm charts

on:
  push:
    tags:
      - "v*.*.*"
    branches:
      - main
      
permissions:
  contents: write

...
```

Reply:
Packaging stable charts now require `permissions/contents: write` while packaging and publishing unstable ones require `permissions/packages: write`.